stdlib/xml/etree/ElementTree.pyi

```diff
@@ -259,11 +260,27 @@ def fromstringlist(sequence: Sequence[str | bytes], parser: XMLParser | None = .
 _ElementFactory = Callable[[Any, Dict[Any, Any]], Element]
 
 class TreeBuilder:
-    def __init__(self, element_factory: _ElementFactory | None = ...) -> None: ...
+    if sys.version_info >= (3, 8):
+        def __init__(
+            self,
+            element_factory: _ElementFactory | None = ...,
+            *,
+            comment_factory: Callable[[str | None], Element] | None = ...,
+            pi_factory: Callable[[str, str | None], Element] | None = ...,
+            insert_comments: bool = ...,
+            insert_pis: bool = ...,
+        ) -> None: ...
+        insert_comments: bool
+        insert_pis: bool
+    else:
+        def __init__(self, element_factory: _ElementFactory | None = ...) -> None: ...
     def close(self) -> Element: ...
     def data(self, __data: str | bytes) -> None: ...
     def start(self, __tag: str | bytes, __attrs: dict[str | bytes, str | bytes]) -> Element: ...
     def end(self, __tag: str | bytes) -> Element: ...
+    if sys.version_info >= (3, 8):
+        def comment(self, __text: str) -> Element: ...
+        def pi(self, __target: str, __text: str | None = ...) -> Element: ...
```

Comment:
Why are these positional-only?

Reply:
I see now, they are positional-only in the C implementation.

Reply:
And now that I think about this for the third time, maybe it would be good to add a comment :)

Reply:
Yup — I had them as pos-or-keyword at first, but stubtest called me out

Reply:
Yes, adding a comment might be good